### DISCUSSION TITLE
create a theme builder page in docs or mantineui ?

### DISCUSSION THREAD

**Discussion by Braggedtooth** [mantinedev/mantine#2008] [Q&A] [answered]
Are there any tools similar to [this ](https://daisyui.com/theme-generator/)for mantine lib
i know [about ](https://mantine.dev/theming/colors/) already just would be nice with a preview of some components while creating a theme

**Comment by rtivital** [comment, marked as answer]
No, currently, we do not provide such feature, maybe it will be added in future releases

**Comment by Braggedtooth** [comment]
Made a quick [proof of concept](https://mantinethemebuilder.netlify.app/) just to see how doable this is, i will continue to work on it when i get time but if anybody is interested let me know here is the [repo](https://github.com/viabay-ab/mtb)
